extensions/default/src/ViewerLayout/index.tsx

```diff
@@ -32,6 +32,8 @@ function ViewerLayout({
   rightPanels = [],
   leftPanelDefaultClosed = false,
   rightPanelDefaultClosed = false,
+  disableHeader = false,
```

Comment:
Can you add a comment that this can be used for mobile use case for instance (since people might think why one wants to remove the toolbar)